specification/vmware/Microsoft.AVS/client.tsp

```diff
@@ -0,0 +1,75 @@
+import "@azure-tools/typespec-client-generator-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+import "./main.tsp";
+
+using Azure.ClientGenerator.Core;
+using Microsoft.AVS;
+
+// client
+@@clientName(Microsoft.AVS, "Avs", "java");
+@@clientName(Microsoft.AVS, "AVS", "python");
+
+// flatten
+#suppress "deprecated" "property flatten for SDK backward compatibility"
+@@flattenProperty(Azure.ResourceManager.TrackedResource.properties);
```

Comment:
need to rename in typespec like https://github.com/Azure/azure-rest-api-specs/blob/061505b89d5d0bbcc9f980e2276f79ff354ce286/specification/vmware/resource-manager/readme.java.md?plain=1#L11-L19